app/connectors/AttachmentsConnector.scala

```diff
@@ -45,4 +45,14 @@ class AttachmentsConnector @Inject()(httpClient: HttpClient, config: FrontendApp
       body = Json.obj("method" -> Json.toJson(attachmentMethod))
     )
 
-}
+  def getIncompleteAttachments(regId: String)(implicit hc: HeaderCarrier): Future[Attachments] = {
```

Comment:
Should just return a List[AttachmentType]

Attachments is a different object that also has the method which this api doesn't deal with

Reply:
hey! thanks, yeah missed in first push. have updated it now to filter out and only send `Ready` docs to FE